### DISCUSSION TITLE
LF 6 MyOpenMath

### DISCUSSION THREAD

**Discussion by kathypinzon** [TeamBasedInquiryLearning/precalculus#372] [General]
Reviewer: @tdegeorge 

1-3 is it a solution?
4-5 graphing
6-9 substitution
10- elimination
11-13 choose best method
14-17 solve (any method)
18-20 solve (infinite, no solution)

**Comment by tdegeorge** [comment]
A couple problems (like 6 and 8) have a message that looks like this: 
<img width="555" alt="Screenshot 2024-07-04 at 12 09 18 PM" src="https://github.com/TeamBasedInquiryLearning/precalculus/assets/143553140/fa12210e-1e3a-4f2e-beff-81abf1209632">
We might want to change these? I am not sure what this means.

**Comment by tdegeorge** [comment]
I added two at the end (problems 21 and 22) which require students to graph to solve. Do we want these too?

**Comment by tdegeorge** [comment]
I kinda wish there were more problems that asked specifically to be solved by elimination (I tried looking and couldn't find any)....I guess it doesn't really matter since there is a mix of problems.
